econml/dml.py

```diff
@@ -333,30 +394,46 @@ class DMLCateEstimator(_DMLCateEstimatorBase):
         If None, the random number generator is the RandomState instance used
         by `np.random`.
 
-    inference: string, inference method, or None
+    inference: string, `Inference` instance, or None
         Method for performing inference.  This estimator supports 'bootstrap'
-        (or an instance of `BootstrapOptions`)
+        (or an instance of `BootstrapInference`) and 'statsmodels' (or an instance of 'StatsModelsInference`)
     """
 
     def __init__(self,
-                 model_y, model_t, model_final=LinearRegression(fit_intercept=False),
+                 model_y=LassoCV(), model_t=LassoCV(),
                  featurizer=PolynomialFeatures(degree=1, include_bias=True),
+                 linear_first_stages=True,
                  discrete_treatment=False,
                  n_splits=2,
-                 random_state=None,
-                 inference=None):
+                 random_state=None):
         super().__init__(model_y=model_y,
                          model_t=model_t,
-                         model_final=model_final,
+                         model_final=StatsModelsWrapper(),
                          featurizer=featurizer,
-                         sparseLinear=False,
+                         linear_first_stages=linear_first_stages,
                          discrete_treatment=discrete_treatment,
                          n_splits=n_splits,
-                         random_state=random_state,
-                         inference=inference)
+                         random_state=random_state)
+
+        self.statsmodelswrapper = self._model_final._model
+
+    def _get_inference_options(self):
+        # add statsmodels to parent's options
+        options = super()._get_inference_options()
+        options.update(statsmodels=StatsModelsInference)
+        return options
 
+    @property
+    def statsmodelsproperties(self):
+        return StatsModelsInference.StatsModelsProperties(self.statsmodelswrapper,
+                                                          self._model_final.effect_op)
+
+    @property
+    def coef_(self):
+        return self.statsmodelswrapper.results.params
 
-class SparseLinearDMLCateEstimator(_DMLCateEstimatorBase):
+
+class SparseLinearDMLCateEstimator(DMLCateEstimator):
     """
     A specialized version of the Double ML estimator for the sparse linear case.
```

Comment:
The docstring should change a bit. Doesn't seem accurate. The sparse linear is the case where the final effect model is high-dimensional sparse linear. Not the first stages.

Reply:
Yes, in the past we conflated the linearity of the first stages with the "SparseLinear" name, which is not ideal.  See also the documentation here: https://econml.azurewebsites.net/spec/estimation/dml.html#sparse-linear-double-ml-estimator which also combines them.  I'll take a stab at improving the docstring.

Reply:
Agreed. We should refactor though all the dml with the new hierarchy we discussed on that meeting. If possible even the documentation. Now  all dml class names only describe the final stage and the first stages could be anything, with defaults set appropriately to what we think is a good default for that final stage.